install/linux/usr/share/odemis/hwtest/zeiss-only.odm.yaml

```diff
@@ -0,0 +1,119 @@
+# For Zeiss Gemini SEM
+# Connection via analog for e-beam scan & serial port for e-beam column settings
+SEM: {
+    class: Microscope,
+    role: sem,
+    children: ["SEM E-beam full", "EBeam Focus", "Sample Stage",
+               "SEM Detector"],
+}
+
+"SEM E-beam full": {
+    class: scanner.CompositedScanner,
+    role: e-beam,
+    dependencies: {
+        external: "SEM E-beam",
+        internal: "EBeam control"
+    },
+#    children: {
+#        "detector": "SEM Detector full",
+#    },
+    init: {},
+    properties: {
+        scale: [8, 8], # (ratio) : start with a pretty fast scan
+        dwellTime: 10.e-6, # s
+    },
+    affects: ["SEM Detector"]
+}
+
+# To use the "external" control from the API instead of the analog
+# Merges the analog and digital detectors (zeiss one and the one from the DAQ board), to provide an "external" signal via the API
+#"SEM Detector full": {
+#    role: se-detector,
+#    dependencies: {
+#        external: "SEM Detector",
+#    },
+#    init: {},
+#}
+
+"SEM Scan Interface": {
+    class: semcomedi.SEMComedi,
+    role: null,
+    init: {device: "/dev/comedi0"},
+    # more detectors can be added, if necessary
+    children: {
+        scanner: "SEM E-beam",
+        detector0: "SEM Detector",
+    }
+}
+
+# Connect:
+# X -> AO 0
+# Y -> AO 1
+# Ground -> AO GND
+"SEM E-beam": {
+    role: null,
+    init: {
+        channels: [0, 1],
+        # On Delmic scanning box v2, the voltage is x2, so need to specify twice smaller values than needed.
```

Comment:
This comment does not seem to be belong to the next line?

Reply:
Good point.